packages/Atoms/Link/TextLink.vue

```diff
@@ -11,47 +12,60 @@
-    text: {default: null, type: String},
-    url: {default: null, type: String},
-    underline: {default: false, type: Boolean},
-    theme: {default: 'light', type: String},
-    size: {default: null, type: String},
-    emphasis: {default: false, type: Boolean}
-  },
-  components: {
-    RplTextIcon,
-    RplLink
-  },
-  computed: {
-    iconSymbolFinal () {
-      if (isExternalUrl(this.url, this.rplOptions.hostname)) {
-        return 'external_link'
+  import {RplTextIcon} from '@dpc-sdp/ripple-icon'
+  import RplLink from './Link.vue'
+  import { isExternalUrl } from '@dpc-sdp/ripple-global/utils/helpers.js'
+
+  export default {
+    name: 'RplTextLink',
+    props: {
+      iconSymbol: {default: null, type: String},
+      iconColor: {default: 'primary', type: String},
+      iconPlacement: {default: 'after', type: String},
+      iconSize: {default: 'm', type: String},
+      text: {default: null, type: String},
+      url: {default: null, type: String},
+      underline: {default: false, type: Boolean},
+      theme: {default: 'light', type: String},
+      size: {default: null, type: String},
+      emphasis: {default: false, type: Boolean}
+    },
+    components: {
+      RplTextIcon,
+      RplLink
+    },
+    computed: {
+      iconSymbolFinal () {
+        if (isExternalUrl(this.url, this.rplOptions.hostname)) {
+          return 'external_link'
+        }
+        return this.iconSymbol
       }
-      return this.iconSymbol
     }
   }
-}
 </script>
 
 <style lang="scss">
   @import "~@dpc-sdp/ripple-global/scss/settings";
   @import "~@dpc-sdp/ripple-global/scss/tools";
 
-  $rpl-text-link-light-underline: rem(4px) solid #E8EBEE !default;
-  $rpl-text-link-light-underline-hover: rem(4px) solid #E8EBEE !default;
-  $rpl-text-link-dark-underline: rem(4px) solid rpl-color('secondary') !default;
-  $rpl-text-link-dark-underline-hover: rem(4px) solid rpl-color('white') !default;
+  $rpl-text-link-light-underline: solid rpl-color('mid_neutral_2');
+  $rpl-text-link-light-underline-hover: solid rpl-color('mid_neutral_2');
+  $rpl-text-link-dark-underline: solid rpl-color('secondary');
+  $rpl-text-link-dark-underline-hover: solid rpl-color('white');
+
+  $rpl-text-link-light-underline-large: rem(3px) $rpl-text-link-light-underline !default;
+  $rpl-text-link-light-underline-small: rem(2px) $rpl-text-link-light-underline !default;
+
+  $rpl-text-link-light-underline-hover-large: rem(3px) $rpl-text-link-light-underline-hover !default;
+  $rpl-text-link-light-underline-hover-small: rem(2px) $rpl-text-link-light-underline-hover !default;
+
+  $rpl-text-link-dark-underline-large: rem(3px) $rpl-text-link-dark-underline !default;
+  $rpl-text-link-dark-underline-small: rem(2px) $rpl-text-link-dark-underline !default;
+
+  $rpl-text-link-dark-underline-hover-large: rem(3px) $rpl-text-link-dark-underline-hover !default;
+  $rpl-text-link-dark-underline-hover-small: rem(2px) $rpl-text-link-dark-underline-hover !default;
```

Comment:
I am ok with these but we can have less code if we split them into `width` and `color` variables and used them in `border-bottom-width` and `border-bottom-color`.